### PR TITLE
映射文档 No. 193/194/195

### PR DESCRIPTION
**Pull request by co63oc**
https://github.com/PaddlePaddle/PaConvert/issues/112

193 torch.distributions.transforms.SoftmaxTransform
194 torch.distributions.transforms.StickBreakingTransform
195 torch.distributions.transforms.StackTransform

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.SoftmaxTransform.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## [torch 参数更多]torch.distributions.transforms.SoftmaxTransform
+
+### [torch.distributions.transforms.SoftmaxTransform](https://pytorch.org/docs/1.13/distributions.html#torch.distributions.transforms.SoftmaxTransform)
```
Comment (Vooolta) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.SoftmaxTransform.md:3]: 这个不要用1.13版本torch的API，要用最新版本torch的API

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StackTransform.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## [torch 参数更多]torch.distributions.transforms.StackTransform
+
+### [torch.distributions.transforms.StackTransform](https://pytorch.org/docs/1.13/distributions.html#torch.distributions.transforms.StackTransform)
```
Comment (Vooolta) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StackTransform.md:3]: 这个不要用1.13版本torch的API，要用最新版本torch的API

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StickBreakingTransform.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+## [torch 参数更多]torch.distributions.transforms.StickBreakingTransform
+
+### [torch.distributions.transforms.StickBreakingTransform](https://pytorch.org/docs/1.13/distributions.html#torch.distributions.transforms.StickBreakingTransform)
```
Comment (Vooolta) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StickBreakingTransform.md:3]: 这个不要用1.13版本torch的API，要用最新版本torch的API

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.SoftmaxTransform.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+paddle.distribution.SoftmaxTransform()
+```
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch    | PaddlePaddle | 备注                                                                       |
+| ---------- | ------------ | -------------------------------------------------------------------------- |
+| cache_size | -            | 表示 cache 大小，Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。 |
```
Comment (ROckDog22) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.SoftmaxTransform.md:21]: 补一下转写示例

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StackTransform.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch    | PaddlePaddle | 备注                                                                       |
+| ---------- | ------------ | -------------------------------------------------------------------------- |
+| tseq       | transforms   | 输入的变换序列，仅参数名不一致。                                           |
+| dim        | axis         | 待变换的轴，仅参数名不一致。                                               |
+| cache_size | -            | 表示 cache 大小，Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。 |
```
Comment (ROckDog22) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StackTransform.md:23]: 补一下转写示例

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StickBreakingTransform.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+paddle.distribution.StickBreakingTransform()
+```
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch    | PaddlePaddle | 备注                                                                       |
+| ---------- | ------------ | -------------------------------------------------------------------------- |
+| cache_size | -            | 表示 cache 大小，Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。 |
```
Comment (ROckDog22) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StickBreakingTransform.md:21]: 补一下转写示例

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.SoftmaxTransform.md (hunk 4 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.SoftmaxTransform.md:21]: https://github.com/PaddlePaddle/docs/blob/9818a5b1e41666513649903449b661a7a818deec/docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.ComposeTransform.md
参考其他文档无转写示例
图片

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StackTransform.md (hunk 5 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StackTransform.md:23]: https://github.com/PaddlePaddle/docs/blob/9818a5b1e41666513649903449b661a7a818deec/docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.ComposeTransform.md
参考其他文档无转写示例
图片

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StickBreakingTransform.md (hunk 6 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.StickBreakingTransform.md:21]: https://github.com/PaddlePaddle/docs/blob/9818a5b1e41666513649903449b661a7a818deec/docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.transforms.ComposeTransform.md
参考其他文档无转写示例
图片

**Review by ROckDog22 — APPROVED** [review]

**Review by zhwesky2010 — APPROVED** [review]
LGTM
